pkg/test/echo/common/response/response.go

```diff
@@ -37,4 +37,5 @@ const (
 	URLField            Field = "URL"
 	HostField           Field = "Host"
 	HostnameField       Field = "Hostname"
+	ClusterField        Field = "Cluster"
```

Comment:
Do we want to make this generic and be "metadata"? Not sure if this is something that would be useful for other applications as well or not - just an idea

Reply:
I'm leaning toward keeping cluster as a top-level property. It seems like a pretty generally useful echo response property.

Reply:
SGTM